base64pp/CMakeLists.txt

```diff
@@ -4,13 +4,7 @@ set(BASE64PP_SOURCES
 set(BASE64PP_INCLUDE_DIR
   ${CMAKE_CURRENT_LIST_DIR}/include)
 
-if (BUILD_SHARED)
-    message(STATUS "Building shared library")
-    add_library(base64pp SHARED ${BASE64PP_SOURCES})
-else()
-    message(STATUS "Building static library")

```

Comment:
Very nice use of CMake's `BUILD_SHARED_LIBS`. Completely forgot about this feature combined with undefined type in `add_library(...)`